packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -1,12 +1,22 @@
 import { ContextualMenuItemType, IContextualMenuItem } from "@fluentui/react";
 import * as React from "react";
 import { useDispatch, useSelector } from "react-redux";
+import { shell } from "electron";
```

Comment:
nit: bff imports are generally alphabetical (tho not enforced by linter), so would go in line 2

Reply:
Where imported from